includes/pricing-details.html

```diff
@@ -43,6 +43,7 @@
           hasAnimation=true
         %}
       {% endfor %}
+      <div class="pt-5">*All prices are excluding VAT. Our offers are addressing professional customers, only.</div>
```

Comment:
Kannst du das in die lang Datei auslagern wie z.B. "tagsClear". Muss dann aber auch in die lang.yml reingepackt werden.